Sources/Fuzzilli/FuzzIL/JSTyper.swift

```diff
@@ -664,6 +664,8 @@ public struct JSTyper: Analyzer {
                 set(instr.output, .boolean)
             case .BitwiseNot:
                 set(instr.output, maybeBigIntOr(.integer))
+            case .Void:
+                set(instr.output, .anything)
```

Comment:
This needs to be `.undefined` I think (the output of the void operator is always `undefined` IIRC)